magda-web-client/src/AppContainer.js

```diff
@@ -23,6 +24,28 @@ import "./AppContainer.css";
 class AppContainer extends React.Component {
     componentDidMount() {
         this.props.requestWhoAmI();
+
+        const _trackers = [];
+
+        // Create GA trackers from the list provided to us
+        config.gapiId
```

Comment:
Should clean this up a little and discuss if better places to put this

Reply:
Yeah I think just refactor it into some kind of `initGoogleAnalytics.js` file that exports a function that does this.

I'm a bit confused by the use of the `?` and the `[config.gapiId]` thing here though - isn't `config.gapiId` already an array?